spec/open_api_spec.rb

```diff
@@ -96,10 +97,37 @@ def random_path
     end
   end
 
+  describe "Parser Validatable attributes" do
```

Comment:
@syncrou Why do we need this?

Reply:
@mkanoor this test loops through our entire spec only and validates that we have the correct attributes applied in any case. If `readOnly`, `nullable`, `enum` isn't set then it validates we have rails validation. If none of them are true it checks that the value is `boolean`.

It's an automatic way to validate we don't forget to add the correct keys to any new or existing Object attributes.

Reply:
@mkanoor though I don't disagree that the name of the context could change.

Reply:
@syncrou I am confused about what the spec is doing and if tis a shared thing that belongs in common. For now its ok

Reply:
@syncrou Suggested we discuss this tomorrow on our call.  For now I agree we can get these changes in and work on improving the logic.  Followup Jira issue: https://projects.engineering.redhat.com/browse/SSP-845